ngx-tools/src/uuid/uuid.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { uuidRegex } from '@terminus/ngx-tools/regex';
+
+import { generateUUID } from './uuid';
+
+
+describe(`uuid`, function() {
+  const testUUIDGenerator = function(generator: () => string, iterations = 1, done: jest.DoneCallback): Promise<void> {
+    const uuidstore: Record<string, string> = {};
+    let i;
+    let newUuid;
+
+    for (i = 0; i < iterations; i++) {
+      newUuid = generator();
+
+      // Test Validity
+      if (!uuidRegex.test(newUuid)) {
+        done.fail(new Error('This is the error'));
```

Comment:
I know it's only a test but the message could be more descriptive: 'UUID is not valid'

Reply:
Whoops.. yeah definitely should make that better.